core/src/impl/Kokkos_CPUDiscovery.cpp

```diff
@@ -46,6 +46,8 @@
 #define KOKKOS_IMPL_PUBLIC_INCLUDE
 #endif
 
+#include <impl/Kokkos_CPUDiscovery.hpp>
+
 #ifdef _WIN32
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
```

Comment:
It would seem that we're not actually doing anything for the Windows case right now.

https://stackoverflow.com/questions/22767620/mingw-missing-sysconf-sc-nprocessors-onln
Has a suggestion for the rough equivalent to what the current code is doing.

Reply:
This is a good point but this would need to be handled elsewhere.

Reply:
Could you please open an issue for that before this gets merged?

Reply:
Done (#5631)